yarn.lock

```diff
@@ -1028,7 +1028,7 @@
 
 "@fortawesome/fontawesome-common-types@^0.2.32":
   version "0.2.32"
-  resolved "https://registry.yarnpkg.com/@fortawesome/fontawesome-common-types/-/fontawesome-common-types-0.2.32.tgz#3436795d5684f22742989bfa08f46f50f516f259"
+  resolved "https://registry.npmjs.org/@fortawesome/fontawesome-common-types/-/fontawesome-common-types-0.2.32.tgz#3436795d5684f22742989bfa08f46f50f516f259"
```

Comment:
Was the change in domain intentional? `yarnpkg.com` => `npmjs.org`

Reply:
Not sure why or when this was changed, but I think this was done on accident somehow probably when I was dealing with problems with yarn, will change it back

Reply:
https://github.com/SaraAlert/SaraAlert/pull/562/commits/18f2cd1c1295b23fca3eb469bf542092217c6acb#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR1031